Ginger/Ginger/Environments/AppDataBasesPage.xaml.cs

```diff
@@ -78,31 +78,34 @@ private void grdMain_PreparingCellForEdit(object sender, DataGridPreparingCellFo
 
         private async void grdMain_CellEditEnding(object sender, DataGridCellEditEndingEventArgs e)
         {
-            if (e.Column.Header.ToString() == "User Password")
+            switch(e.Column.DisplayIndex) //we have checked for column index for grid
             {
-                Database selectedEnvDB = (Database)grdAppDbs.CurrentItem;
-                String intialValue = selectedEnvDB.Pass;
-                //if Pass is stored in the form of variable, encryption not required at this stage
-                if (!string.IsNullOrEmpty(intialValue) && !intialValue.Contains("{Var Name"))
-                {
-                    if (!EncryptionHandler.IsStringEncrypted(intialValue))
+                case 8:
```

Comment:
Please add comment which explains what case 8 and case 0 is for

Reply:
Done